plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/ForteFBTemplate.xtend

```diff
@@ -379,31 +379,34 @@ abstract class ForteFBTemplate<T extends FBType> extends ForteLibraryElementTemp
 
 	def protected generateSetInitialValuesDeclaration(Iterable<VarDeclaration> variables) '''
 		«IF !variables.empty»
-			void setInitialValues(bool retain) override;
+			void setInitialValues(bool cold) override;
 		«ENDIF»
 	'''
 
 	def protected generateSetInitialValuesDefinition(Iterable<VarDeclaration> variables) '''
 		«IF !variables.empty»
-			void «className»::setInitialValues(bool retain) {
+			void «className»::setInitialValues(bool cold) {
```

Comment:
I missed that yesterday but can you please move the `cold) {`  part into the if and else, and in the else only add `){`

Because if there are no retain variables the cold param is not needed. 

And as we are at it: In 4diac FORTE we have the coding guideline that function parameters stat with pa. So please change `cold` to `paCold`.